api/v1alpha1/releaseplan_types.go

```diff
@@ -17,60 +17,55 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"github.com/redhat-appstudio/release-service/kcp"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// ReleaseLinkSpec defines the desired state of ReleaseLink.
-type ReleaseLinkSpec struct {
-	// DisplayName refers to the name of the ReleaseLink to link a user and managed workspace together
-	// +required
+// ReleasePlanSpec defines the desired state of ReleasePlan.
+type ReleasePlanSpec struct {
+	// DisplayName is the long name of the ReleasePlan
+	// +optional
 	DisplayName string `json:"displayName"`
 
 	// Application is a reference to the application to be released in the managed workspace
 	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
 	// +required
 	Application string `json:"application"`
 
-	// Target is a reference to the workspace to establish a link with
-	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
+	// Target references where in the cluster to find the information required for a release
```

Comment:
Much like origin, this comment is not useful to me at all

Reply:
Changed to `// Target references where to send the release requests`.

Reply:
I like that a lot better. I don't *love* it but I have no better suggestions